src/service/get-words.js

```diff
@@ -1,8 +1,9 @@
+import { httpService } from './http-service';
 import { WORDS_COUNT } from '../config/config';
```

Comment:
right. additionally I just noticed that in `config.js` I'm importing `{ words } from '../data/data';` instead of refering to our new database.
I'll fix this.